tests/selectors/test-app.selectors.ts

```diff
@@ -1,6 +1,5 @@
 export enum TestAppSelectors {
+  BtnContractCall = 'btn-contract-call',
   BtnUpdateValidProfile = 'btn-update-valid-profile',
   BtnUpdateInvalidProfile = 'btn-update-invalid-profile',
```

Comment:
hm, so may be its better to move `BtnUpdateValidProfile` and `BtnUpdateInvalidProfile` to `UpdateProfileRequestSelectors` ?

Reply:
Those actually live in the test-app code, so I was going to leave those.